os-config_arm.yml

```diff
@@ -0,0 +1,328 @@
+hostname: rancher
```

Comment:
I'm going to lose sleep over the fact that this file is 90% the same as `os-config.yml`.   Do we have the technology to not duplicate so much?